packages/zoe/src/types-ambient.js

```diff
@@ -13,26 +13,42 @@
 /**
  * @typedef {string} Keyword
  * @typedef {Handle<'Invitation'>} InvitationHandle - an opaque handle for an invitation
- * @typedef {Record<Keyword, Issuer<any>>} IssuerKeywordRecord
- * @typedef {Record<Keyword, ERef<Issuer<any>>>} IssuerPKeywordRecord
- * @typedef {Record<Keyword, Brand<any>>} BrandKeywordRecord
```

Comment:
these types should be maintained. 

when there is a more specific type, it's a concrete type that extends this more general one, not a parameterization of this.